src/devices/ikea.ts

```diff
@@ -58,7 +58,7 @@ const bulbOnEvent: OnEvent = async (type, data, device, options, state: KeyValue
             if (onLevel > 255) onLevel = 254;
```

Comment:
Hope you don't mind me asking, why would onLevel be more than 255 and why isn't it clamped to 255 instead?

Also, under what circumstance(s) would `Number(onLevelRaw)` return a number greater than 255?